ci/travis-install.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 set -ev
 
-if [ "$TRAVIS_NODE_VERSION" == "v0.12.17" ]; then 
-    npm install --force && npm link && npm link webpack;
-else 
-    npm install yarn -g && yarn install && yarn link || true && yarn link webpack;
-fi
+npm install yarn -g && yarn install && yarn link || true && yarn link webpack;
```

Comment:
According to this [blog post](https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn), it only gets installed when you have a `yarn.lock` file, which we don't have.